src/Microsoft.TestPlatform.Utilities/XmlUtilities.cs

```diff
@@ -5,7 +5,9 @@ namespace Microsoft.VisualStudio.TestPlatform.Utilities
 {
     using System;
     using System.Diagnostics.CodeAnalysis;
+#if NET451
```

Comment:
why? Is this suggested by some tool?

Reply:
SecurityElement.Escape(innerXml); uses this library only so a tool SonarQube suggested to create a block for NET451 the using doesn't apply to NETCORE2.0. im not sure if this is an anti pattern though. Let me know if you want to revert it.